content/lua/concepts/Comments/comments.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Comments' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A comment is a piece of text within a program that is not being executed. It can provide additional information to support understanding the code.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Comments'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+
+## Single-line Comments
```

Comment:
The entry should begin with the description. Sometimes the description used in the metadata section is expanded here if there's additional context or content that is valuable to include.

Reply:
Yup ^ The description from the metadata can likely just be pulled down. I'll add as a suggestion.